onnxscript/rewriter/ort_fusions/_core.py

```diff
@@ -130,7 +129,7 @@ def optimize_for_ort(
         - The optimized `ir.Model` after applying transformer-specific fusions.
         - A dictionary with a count of each of the fusions applied.
     """
-
+    rewrite(model, [gemm_to_matmul_add.rule])
```

Comment:
I think we can add the rule to the list of rewrite-rules at the top of the file, which already has other rules.

Reply:
Does any of the xformer fusion rule depend on it? Currently the list of rewrite rules are run after the xformer fusion rules

Reply:
Good question. Possible. The fusions do involve MatMul.